contracts/axelar-gateway/src/contract.rs

```diff
@@ -223,6 +223,16 @@ impl AxelarGatewayInterface for AxelarGateway {
     fn epoch(env: &Env) -> u64 {
         auth::epoch(env)
     }
+
+    fn version(env: Env) -> Vec<u32> {
+        vec![&env, 0, 0, 1]
+    }
+
+    fn upgrade(env: Env, new_wasm_hash: BytesN<32>) {
+        Self::operator(&env).require_auth();
```

Comment:
we need to add a separate owner role, similar to EVM gateway. can add in a separate PR though